mapie_v1/integration_tests/tests/test_regression.py

```diff
@@ -1,9 +1,334 @@
+from __future__ import annotations
+
 import numpy as np
-from sklearn.model_selection import train_test_split
+import pytest
+from numpy.testing import assert_almost_equal
+from sklearn.datasets import make_regression
+from sklearn.linear_model import LinearRegression
+from sklearn.ensemble import RandomForestRegressor
+
+from mapie_v1.regression import (
+    SplitConformalRegressor,
+    CrossConformalRegressor,
+    JackknifeAfterBootstrapRegressor,
+    ConformalizedQuantileRegressor
+)
+from mapiev0.regression import MapieRegressor as MapieRegressorV0 # noqa
+from mapiev0.regression import MapieQuantileRegressor as MapieQuantileRegressorV0 # noqa
+from mapie_v1.conformity_scores.utils import select_conformity_score
+from mapie_v1.utils import (filter_params,
+                            calculate_coverage,
+                            train_test_split_shuffle)
+from sklearn.model_selection import KFold
+
+RANDOM_STATE = 1
+K_FOLDS = 3
+N_BOOTSTRAPS = 30
+
+X_toy = np.array([0, 1, 2, 3, 4, 5]).reshape(-1, 1)
+y_toy = np.array([5, 7, 9, 11, 13, 15])
+X, y = make_regression(n_samples=500,
+                       n_features=10,
+                       noise=1.0,
+                       random_state=RANDOM_STATE)
+
+
+@pytest.mark.parametrize("strategy_key", ["split"])
+@pytest.mark.parametrize("method", ["base", "plus", "minmax"])
+@pytest.mark.parametrize("conformity_score", ["absolute"])
+@pytest.mark.parametrize("confidence_level", [0.9, 0.95, 0.99])
+@pytest.mark.parametrize("agg_function", ["mean", "median"])
+@pytest.mark.parametrize("minimize_interval_width", [True, False])
+@pytest.mark.parametrize("allow_infinite_bounds", [True, False])
+@pytest.mark.parametrize(
+    "estimator", [
+        LinearRegression(),
+        RandomForestRegressor(random_state=RANDOM_STATE, max_depth=2)])
+@pytest.mark.parametrize("test_size", [0.2, 0.5])
+def test_exact_interval_equality(
```

Comment:
Let's have 1 test for each v1 class maybe, so that one would be `test_exact_interval_equality_for_split`